apps/web-giddh/src/app/verify-subscription-transfer-ownership/verify-subscription-transfer-ownership.component.ts

```diff
@@ -50,12 +61,22 @@ export class VerifySubscriptionTransferOwnershipComponent implements OnInit {
             }
         });
 
+
         this.dialogOpen();
+        this.rejectForm = this.fb.group({
+            reason: ['']
+        });
 
-        this.verifyOwnershipSuccess$.pipe(takeUntil(this.destroyed$)).subscribe(response => {
-            if (response) {
+        combineLatest([
+            this.verifyOwnershipSuccess$.pipe(takeUntil(this.destroyed$)),
+            this.rejectReason$.pipe(takeUntil(this.destroyed$))
+        ]).subscribe(([verified, reject]) => {
+            if (verified && !reject?.reqId) {
                 this.acceptedSubscription = true;
-                this.modalDialogRef.close();
+                this.dialog?.closeAll();
```

Comment:
Write this after else block because it call in both if and else block

Reply:
DONE